services/frontend-v3/src/components/resultsCard/ResultsCardView.jsx

```diff
@@ -133,7 +133,7 @@ const ResultsCardView = ({
     }
 
     if (person.groupLevel) {
-      subtitle = subtitle + " (" + person.groupLevel.name + ")";
+      subtitle = subtitle.concat(" (", person.groupLevel.name, ")");
```

Comment:
I think string interpolation is used more with back ticks in the code, maybe changing it to this?

```suggestion
      subtitle = `${subtitle} (${person.groupLevel.name})`;
```

Reply:
on it! 

I will leave this link for future reference:
https://stackoverflow.com/questions/27565056/es6-template-literals-vs-concatenated-strings/27567436